README.md

```diff
@@ -1,155 +1,66 @@
-```
-
-`charmcraft pack` will fetch additional files into the tree from PyPI based
-on `requirements.txt` and will compile modules using a virtualenv.
-
-The charmed operator is just a zipfile with metadata and the operator code
-itself:
-
-```text
-$ unzip -l test-charm.charm
-Archive:  test-charm.charm
-  Length      Date    Time    Name
----------  ---------- -----   ----
-      221  2020-11-15 08:10   metadata.yaml
-[...]
-    25304  2020-11-15 08:14   venv/yaml/__pycache__/scanner.cpython-38.pyc
----------                     -------
-   812617                     84 files
+# Use the Multipass 'charm-dev' blueprint to quickly get an Ubuntu VM with Charmcraft
+# as well as everything else you’ll need to develop and test a charm,
+# including `tox`, a Kubernetes cloud -- MicroK8s -- and Juju.
+# Bootstrap a Juju controller into the cloud and create a Juju model to deploy to.
+sudo snap install multipass && \
+multipass launch --cpus 4 --memory 8G --disk 30G --name tutorial-vm charm-dev
+juju bootstrap microk8s tutorial-controller; juju add-model tutorial-model
+
+# Create a charm directory and use Charmcraft to initialise your charm file structure:
+mkdir my-new-charm; cd my-new-charm; charmcraft init
+
+# The files are already populated with functional templates. Poke around.
+# Note that the ‘metadata.yaml’ shows what we have is an example charm
+# called ‘my-new-charm’ which uses an OCI image resource ‘httpbin’ from ‘kennethreitz/httpbin’.
+
+# Pack the example charm into a .charm file:
+charmcraft pack
+
+# This yields a file called ‘my-new-charm_ubuntu-22.04-amd64.charm’
+# (the architecture bit may be different depending on your architecture).
+# Use the name from the packing step
+# and the resource from the ‘metadata.yaml’
+# to deploy your example charm to your local MicroK8s cloud:
+juju deploy ./my-new-charm_ubuntu-22.04-amd64.charm --resource httpbin-image=kennethreitz/httpbin
+
+# Congratulations, you’ve just built your first charm!
+
+# Now you can go back and edit the files to make the charm your own.
+# Once you’re done, come back to Charmcraft to publish and release your charm on Charmhub!
+
+# Clean up:
+# - delete the vm:
+multipass delete --purge my-vm
+# - uninstall multipass:
+snap remove multipass
 ```
+For more, see our user docs: https://juju.is/docs/sdk 
 
-Now, if you have a Kubernetes cluster with the Juju OLM accessible you can issue
-`juju deploy ./my-new-charm.charm --resource httpbin-image=kennethreitz/httpbin`.
-For classic machine deployments you can issue `juju deploy ./my-new-charm.charm`.
-You do not need to publish your operator on Charmhub, you can pass the charmed
-operator file around directly to users, or for CI/CD purposes.
-
-## Charmhub login and charm name reservations
+## Contribute to Charmcraft
+Read our [Code of conduct](https://ubuntu.com/community/code-of-conduct) and start chatting with us:
+- Join our chat: [Mattermost](https://chat.charmhub.io/charmhub/channels/charm-dev) 
```

Comment:
we have a specific charmcraft channel

Reply:
I know. I chose the Charm Development channel because it seemed to be the more popular entry point. As in, a user consumes Charmcraft through the SDK, whose main Mattermost channel is the Charm development channel. But I agree -- we either update that general channel to provide links to the more specific channels, including Charmcraft, or we link directly to the Charmcraft channel. What do you think?

Reply:
I changed it to the charmcraft channel.